docs/reference/configuration-reference/alerting-settings.md

```diff
@@ -134,6 +134,23 @@ $$$action-config-email-domain-allowlist$$$
 
     Data type: `string`
 
+`xpack.actions.email.recipient_allowlist` ![logo cloud](https://doc-icons.s3.us-east-2.amazonaws.com/logo_cloud.svg "Supported on {{ech}}")
```

Comment:
```suggestion
`xpack.actions.email.recipient_allowlist` ![logo cloud](https://doc-icons.s3.us-east-2.amazonaws.com/logo_cloud.svg "Supported on {{ech}}") {applies_to}`stack: ga 9.2`
```
In case this isn't overwritten by the settings-gen source file content yet, adding a tag to indicate that this is available from 9.2 (see my comment in the settings-gen yml file for context.